misc/scripts/library-coverage/create-pr.py

```diff
@@ -0,0 +1,112 @@
+import sys
+import settings
+import utils
+import shutil
+import json
+import os
+
+"""
+This script compares the generated CSV coverage files with the ones in the codebase. And if they are different, it
+creates a PR with the changes. If a PR already exists, then it will be updated to reflect the latest changes.
+"""
+
+# Name of the branch that's used to push the updated coverage files, also the head of the PR
+branch_name = "workflow/coverage/update"
```

Comment:
Does this have to be unique? If so include the run or PR number?

Reply:
We're using a single dedicated branch for these. At each run of the workflow, we force push the current state of `main` and additionally a commit with the new coverage reports.